src/modules/accommodations/entities/accommodation.entity.ts

```diff
@@ -62,8 +62,8 @@ export class Accommodation {
   @ManyToOne(() => AccommodationCategory, (category) => category.accommodations)
   category: AccommodationCategory;
 
-  @OneToMany(() => AccommodationLike, (like) => like.accommodation)
-  likes?: AccommodationLike[];
+  @OneToMany(() => Wishlist, (like) => like.accommodation)
```

Comment:
why this is one to many instead of m-to-m? you can do something like this:  
```ts
@ManyToMany(() => Wishlist, wishlist => wishlist.accommodations)
    wishlists: Wishlist[];
```

Reply:
One user can have more wishlists, but one wishlist can not have more than one user, That is why it is one to many, I created another table, not make relation user-to-accommodation directly

Reply:
this is an Accommodation Entity...

Reply:
actually I did it on user module, but Muhammadjon teacher said to implement it on accommodation, not in user. So I changed it to accommodation

Reply:
Begzod, You had this idea, why did not you tell about it yesterdays meeting. We discussed about how to implement it and decided to create table for wishlist, now I need to rewrite it fully again. But it is working correct, I think no need for this

Reply:
we have had this idea all along...